src/Hound.java

```diff
@@ -0,0 +1,83 @@
+
+
+
+public class Hound {
+	private int x_location;
```

Comment:
All the code below is duplicated between classes, which is redundant. You can refactor it by moving this code into a parent superclass that both classes extend.